lib/nexmo_developer/app/helpers/changelogs_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ChangelogsHelper
+  def build_volta_icon(element)
+    return '' unless element
+
+    if element.scan(/vonage-(\w+)-sdk/).present?
+      element.scan(/vonage-(\w+)-sdk/).flatten.first.downcase
+    elsif element.scan(/(\w+) SDK/).present?
+      element.scan(/(\w+) SDK/).flatten.first.downcase
+    elsif element.scan(/(\w+)-cli/).present?
+      element.scan(/(\w+)-cli/).flatten.first.downcase
+    end
```

Comment:
This conditional logic could perhaps be simplified/ eliminated with a different approach or alternative regex pattern